src/jit/function-builder.cc

```diff
@@ -391,6 +455,26 @@ bool FunctionBuilder::Emit(TR::BytecodeBuilder* b,
       break;
     }
 
+    case Opcode::Br: {
+      auto target = &thread_->GetIstream()[ReadU32(&pc)];
```

Comment:
No need to call `thread_->GetIstream()` here, the correct value is already in the `istream` parameter (same problem in `InterpBrUnless`).